src/themes/save-on-energy/theme.json

```diff
@@ -695,10 +695,33 @@
         "display": "block",
         ":not(:first-child)": {
           "mt": "sm"
+        },
+        "variants": {
+          "calculator": {
+            "variant": "styles.h6",
+            "fontWeight": "bold",
+            "color": "grey-80",
+            "display": "block",
+            "mb": "xs",
+            ":not(:first-child)": {
```

Comment:
have you browser checked this? https://caniuse.com/#search=%3Anot() I dont think not is well supported at all

Reply:
might be fine actually as your using it as a negation, but just want to check

Reply:
It does actually appear to be fine in IE11, not sure why caniuse is saying it wouldnt

Reply:
yeah I would never use `:not`. You should be able to do the same thing by giving `calculator` the `"mt": "xl"` and then doing `:first-child: { mt: 0 }`